src/main/resources/pl/edu/agh/iet/dts/ui/static/js/controller/userFormController.js

```diff
@@ -2,6 +2,9 @@ app.controller('userFormController', ['$http', '$scope', '$rootScope', '$interva
 
     $scope.submitForm = function () {
         $('#form-panel').css('display', 'none');
+        
+        $interval.cancel($rootScope.promise);
+        $rootScope.promise = $interval($rootScope.locationRequest, $rootScope.aggregationTime * 60 * 1000);
```

Comment:
The same code should be invoked at the startup of the application, when a user logs in. Why? Imagine a situation, when a user logs in, sets the preferences and logs out. When a user logs in again, he/she should have this daemon started without updating preferences again.